### PR TITLE
my new request

### PR DESCRIPTION
**Pull request by ronshif**

### CODE REVIEW

File: ConsoleApp2/Program.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Program
     {
         static void Main(string[] args)
         {
-             //b5
+             //b2
```
Comment (ronshif) [inline ConsoleApp2/Program.cs]: this is no good

File: ConsoleApp2/Program.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ class Program
...
             //Vs_ronshif
 
             Runner run = new Runner();
```
Comment (ronshif) [inline ConsoleApp2/Program.cs]: ddd

**Review by ronshif — COMMENTED** [review]
sdfs

File: ConsoleApp2/Program.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@ class Program
     {
         static void Main(string[] args)
         {
```
Comment (shifroma) [inline ConsoleApp2/Program.cs]: ddssdf

**Review by shifroma — APPROVED** [review]

**Pull request merged**